resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <div>
+    <b-container fluid>
+      <h3>
+        {{ $t('settings.application.title') }}
+        <b-button id="application-save-button"
+                  class="float-right"
+                  variant="success"
+                  @click="updateSettings(settings)"
+                  :disabled="isBusy">
+          <span><i class="fas fa-save mr-2"></i>{{ $t('settings.application.save') }}</span>
+        </b-button>
+      </h3>
+
+      <hr>
+
+      <!--Logo Settings-->
+      <b-form-group
+        label-for="application-logo-input"
+        :description="$t('settings.application.logo.description')"
+        :state='fieldState("logo")'
+      >
+
+        <template v-slot:label>
+          <span><i class="fas fa-camera-retro mr-3"></i></span>
+          {{ $t('settings.application.logo.title') }}
+        </template>
+
+        <b-row align-v="baseline" class="my-3">
+          <b-col sm="6" lg="3" class="text-center">
+            <b-img
+              :src="settings.logo"
+              class="my-2"
+              rounded="0"
+              alt="application-logo-preview"
+              width="150"
+              height="100"
+              fluid
+            >
+            </b-img>
+          </b-col>
+          <b-col sm="6" lg="9">
+            <b-input-group>
+              <b-form-input id="application-logo-input"
```

Comment:
Please give the user the option to set the logo url or upload a png or jpeg file as logo, store the file in the public storage and set path accordingly